recipes/kartothek/LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2019 JDA Software, Inc
```

Comment:
LICENSE is included upstream in the next release: https://github.com/JDASoftwareGroup/kartothek/pull/11